packages/wast-parser/test/tokenizer/number-literal-errors/multiple-colons/actual.wast

```diff
@@ -1 +0,0 @@
-nan::1234
```

Comment:
Are these tests not possible anymore?

Reply:
Ahh not sure why these are removed, I'll adjust and add them back in. Sorry

Reply:
I restored them 7095053.